src/server.c

```diff
@@ -6182,8 +6182,7 @@ void addSectionsToDict(dict *section_dict, char **sections, int len) {
     }
 }
 
-
-dict *genInfoSectionDict(client *c, const char *source, int *all_sections, int *everything) {
+dict *genInfoSectionDict(char **argv, int argc, int *out_all, int *out_everything) {
```

Comment:
how come this is the only change n that function?
the code below still refers to `c`?
i now realize you pushed a partial commit that doesn't work.
i'll stop the review. let me know when ready.

Reply:
@oranagra  Now codes are changed based on our discussion a few days ago. And CI is passed. Please take sometime to take a look.  Thanks